### PR TITLE
Download Crowdin resources periodically/manually

### PR DESCRIPTION
**Pull request by thc202**
Add workflows to allow to build the translations packages and download
the localized resources periodically (twice a week) and manually.
Update build to support the above workflows.
The localized resources will be copied (pull request) to the
corresponding repositories (as defined in the configuration file
`crowdin-tasks.yml`).

### CODE REVIEW

**Comment by thc202** [comment]
Set to WIP as we still need to upload the latest resources to Crowdin. Changes should be ready to review though.

**Review by psiinon — APPROVED** [review]

File: .github/workflows/crowdin-build-translations.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+  build:
+    name: Build Translations
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Java
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
```
Comment (kingthorin) [inline .github/workflows/crowdin-build-translations.yml:17]: Build with 8 but download with 11?

**Review by kingthorin — APPROVED** [review]
Looks good to me

File: .github/workflows/crowdin-build-translations.yml (hunk 1 above)
Comment (kingthorin) [reply .github/workflows/crowdin-build-translations.yml:17]: I guess that makes sense as we're still building 8 but may as well use 11 for non-build stuff. Less to change later.

Comment (thc202) [reply .github/workflows/crowdin-build-translations.yml:17]: Build uses just this repo, download/copy uses all of them, picked 11 for the later just to be safe (although 8 should work just fine).

**Comment by thc202** [comment]
No longer WIP.

**Comment by kingthorin** [comment]
Still looks good to me.

**Pull request merged**